<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CInductiveBookWidget</class>
 <widget class="QWidget" name="CInductiveBookWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>912</width>
    <height>718</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>归纳本</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="CListViewEx" name="listView">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>150</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer_2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLineEdit" name="lineEdit">
       <property name="placeholderText">
        <string>请输入文本标题</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer_3">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="CCheckButton" name="checkButton" native="true"/>
     </item>
     <item>
      <widget class="CLSwitchButton" name="switchButton">
       <property name="textOff">
        <string>显示全文</string>
       </property>
       <property name="textOn">
        <string>显示归纳</string>
       </property>
       <property name="checked">
        <bool>false</bool>
       </property>
       <property name="buttonStyle">
        <enum>CLSwitchButton::EBS_CircleIn</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnOcr">
       <property name="text">
        <string>识图</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnSave">
       <property name="text">
        <string>保存</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QTextEdit" name="textEdit"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CLSwitchButton</class>
   <extends>QWidget</extends>
   <header>LxWidgets/LSwitchButton.h</header>
  </customwidget>
  <customwidget>
   <class>CListViewEx</class>
   <extends>QListView</extends>
   <header>ListViewEx.h</header>
  </customwidget>
  <customwidget>
   <class>CCheckButton</class>
   <extends>QWidget</extends>
   <header>CheckButton.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
